site/cypress/e2e/sidebar.spec.js

```diff
@@ -0,0 +1,102 @@
+import { getCurrentSemester } from '../support/utils.js';
+
+const currentSemester = `${getCurrentSemester()}`;
+
+Cypress.Commands.add('sidebarContains', (title, extension, header = title) => {
```

Comment:
You don't have to change this but just as a heads for for future cases you can use simple js `function`s to group similar behavior since you'll still have access to the `cy` object. You can also return the result of a `cy` action to chain the result in the caller.

Typically commands are used for test suite wide logic since you can import and use them wherever